src/main/java/frc/robot/commands/intakeCommands/IntakeMoveCommand.java

```diff
@@ -16,12 +17,18 @@ public class IntakeMoveCommand extends Command {
 
   double setPoint;
   boolean toggleEnable;
+  int slot;
+  double feedForward;
+  Timer timer = new Timer();
```

Comment:
Nit: All of these fields should probably be private final. instead of package private.